README.md

```diff
@@ -105,3 +105,9 @@ Pull requests are welcome! For major changes, please open an issue first to disc
 ## License
 
 This project is licensed under the MIT License.
+
+## Environment Variables
```

Comment:
"Environment variables" section should appear before "Running the application" section "